src/patternfly/components/Table/table-tree-view.scss

```diff
@@ -78,3 +78,176 @@ $pf-c-tree-view--MaxDepth: 10;
   min-width: var(--pf-c-table__tree-view-icon--MinWidth);
   margin-right: var(--pf-c-table__tree-view-icon--MarginRight);
 }
+
+.pf-c-table__tree-view-details-toggle {
+  display: none;
+  visibility: hidden;
+}
+
+// stylelint-disable
+@mixin pf-tree-view-mobile-layout {
+  .pf-m-tree-view-grid.pf-c-table {
+    @content;
+  }
+
+  .pf-m-tree-view-grid-md.pf-c-table {
+    @media screen and (max-width: $pf-global--breakpoint--md) {
+      @content;
+    }
+  }
+
+  .pf-m-tree-view-grid-lg.pf-c-table {
+    @media screen and (max-width: $pf-global--breakpoint--lg) {
+      @content;
+    }
+  }
+
+  .pf-m-tree-view-grid-xl.pf-c-table {
+    @media screen and (max-width: $pf-global--breakpoint--xl) {
+      @content;
+    }
+  }
+
+  .pf-m-tree-view-grid-2xl.pf-c-table {
+    @media screen and (max-width: $pf-global--breakpoint--2xl) {
+      @content;
+    }
+  }
+}
+
+@include pf-tree-view-mobile-layout {
+  --pf-c-table--pf-m-tree-view-mobile--tr--OutlineOffset: calc(-1 * var(--pf-global--spacer--xs));
```

Comment:
should any `pf-c-table--pf-m-tree-view-mobile` var be `pf-c-table--pf-m-tree-view-grid`?